src/App.js

```diff
@@ -1,25 +1,52 @@
-import logo from './logo.svg';
 import './App.css';
+import React, {useState} from 'react';
+import foods from './foods.json';
+import FoodBox from './components/FoodBox'
+import AddFoodForm from './components/AddFoodForm'
+import Search from './components/Search'
 
 function App() {
+  const [dishes, setDishes] = useState(foods)
+
```

Comment:
For the Search function to work properly you need to add a new useState:
`  const [showFoods, setShowFoods] = useState(foods);`